chirp/wxui/memedit.py

```diff
@@ -872,12 +869,13 @@ def filter_unknowns(items):
         valid_modes = filter_unknowns(self._features.valid_modes)
         valid_skips = filter_unknowns(self._features.valid_skips)
         valid_duplexes = filter_unknowns(self._features.valid_duplexes)
-        valid_power_levels = self._features.valid_power_levels
         valid_tuning_steps = self._features.valid_tuning_steps
         if self._features.has_variable_power:
-            power_col_cls = ChirpVariablePowerColumn
+            power_column = ChirpVariablePowerColumn('power', self._radio)
         else:
-            power_col_cls = ChirpChoiceColumn
+            valid_power_levels = self._features.valid_power_levels
+            power_column = ChirpChoiceColumn('power', self._radio,
+                                             valid_power_levels)
```

Comment:
This too. I might should have done it this way from the beginning, but I don't think these sorts of changes are worth the churn, personally.